generated/api/v1/network_graph_service.swagger.json

```diff
@@ -496,7 +496,8 @@
         },
         "dstPort": {
           "type": "integer",
-          "format": "int64"
+          "format": "int64",
+          "description": "may be 0 if not applicable (e.g., icmp)."
```

Comment:
any idea why this was changed in this PR?

Reply:
Long story, style is failing without because of an unrelated other PR merged yesterday. Might not be needed after rebase (assuming that was fixed)